cmd/dockerd/service_windows.go

```diff
@@ -409,6 +410,12 @@ func initPanicFile(path string) error {
 		return err
 	}
 
+	// Reset os.Stderr to the panic file (so fmt.Fprintf(os.Stderr,...) actually gets redirected)
+	os.Stderr = os.NewFile(uintptr(panicFile.Fd()), "/dev/stderr")
```

Comment:
If someone is questioning why a *nix-style path, it's copied right out of how golang initializes os.Stderr